src/status_im/utils/handlers.cljs

```diff
@@ -103,9 +103,8 @@
                anon-id  (ethereum/sha3 current-account-id)]
            (doseq [{:keys [label properties]}
                    (mixpanel/matching-events new-db event mixpanel/event-by-trigger)]
-             (mixpanel/track anon-id label properties offline?))
-           (when (= :send-current-message (first event))
-             (instabug/maybe-show-survey new-db)))))
+             (mixpanel/track anon-id label properties offline?)
+             (instabug/track label properties)))))
```

Comment:
We have a queue for mixpanel events (which stores events while app is offline), do we need the same fo instabug?

Reply:
@rasom not necessary, it's okay to loose some of instabug events as they only used to show surveys